src/status_im2/navigation/screens.cljs

```diff
@@ -232,7 +239,28 @@
      :options   {:theme      :dark
                  :layout     options/onboarding-transparent-layout
                  :animations transitions/push-animations-for-transparent-background}
-     :component welcome/view}]
+     :component welcome/view}
+
+    {:name      :wallet-accounts
+     :component wallet-accounts/view}
+
+    {:name      :wallet-collectibles
+     :component wallet-collectibles/view}
+
+    {:name      :wallet-create-account
+     :component wallet-create-account/view}
+
+    {:name      :wallet-saved-address
+     :component wallet-saved-address/view}
+
+    {:name      :wallet-saved-addresses
+     :component wallet-saved-addresses/view}
+
+    {:name      :wallet-send
+     :component wallet-send/view}
+
+    {:name      :wallet-main
+     :component wallet-main/view}]
```

Comment:
We won't be using this as the `wallet-main` is part of the bottom tab.

Reply:
Good point, will remove :)